interpretation/deepseismic_interpretation/dutchf3/data.py

```diff
@@ -712,6 +731,13 @@ def __getitem__(self, index):
             im, lbl = augmented_dict["image"], augmented_dict["mask"]
             im = _transform_HWC_to_CHW(im)
 
+        # dump images and labels to disk
+        if self.debug:
+            outdir = f"patchLoader_{self.split}_{'aug' if self.augmentations is not None else 'noaug'}"
+            generate_path(outdir)
+            mask_to_disk(im[0,:,:], f"{outdir}/{index}_img.png")
+            mask_to_disk(lbl, f"{outdir}/{index}_lbl.png")
```

Comment:
same comment as above on mask_to_disk

Reply:
same as above - thanks for the feedback!